src/components/App.tsx

```diff
@@ -14,7 +15,9 @@ const App: React.FunctionComponent = () => {
       <Routes>
         <Route path="/" element={<LandingPage />} />
         <Route path="/fun-facts" element={<FunFactPage />} />
-        <Route path="/rovers" element={<RoverSelectorPage />} />
+        <Route path="/rovers" element={<RoverSelectorPage />}/>
+        <Route path="/curiosity" element={<CuriosityMap />}/>
```

Comment:
Why does the `CuriosityMap` have its own route? I thought it was a sub-component that would appear on a different page? Let's talk about this